Tests/Swallow/DebugLogMethodMacroTests.swift

```diff
@@ -0,0 +1,165 @@
+//
+// Copyright (c) Vatsal Manot
+//
+
+import SwallowMacros
+import SwallowMacrosClient
+import SwiftSyntaxMacrosTestSupport
+import XCTest
+
+final class DebugLogMethodMacroTests: XCTestCase {
+    func testDebugLogMethodMacro() {
+        let testClass = TestClass()
+        // Should print
+        testClass.debugTestOne(x: 10, y: "20")
+        // Shouldn't print
+        testClass.debugTestTwo(x: 20, y: "40")
+        // Should print
+        let z = TestStruct(a: 1, b: "New", c: CGPoint(x: 10, y: 10))
+        let _ = testClass.debugTestThree(x: 0, y: "20", z: z)
+    }
+    
```

Comment:
`_DebugLogMethod` can be made into a static variable called `macroNameIdentifier`